.github/workflows/changelog.yml

```diff
@@ -49,7 +49,7 @@ jobs:
             theming:
               - 'packages/theming/**'
             plone7:
-              - 'packages/revolto/**'
+              - 'packages/voltotng/**'
```

Comment:
How do you pronounce that? 😝 

I still like "DEVOlto" better. Are we not men? We are DEVOlto!

Reply:
@stevepiercy tip: you don't, because you'll never need to pronounce it. ;) We are bad ass trekkies, at most :)

Reply:
@sneridagh looks like Klingon to me.

Reply:
PloneNG2 :laughing:

Reply:
😂